scripts/check_imports.swift

```diff
@@ -57,19 +57,24 @@ let skipImportPatterns = [
   "FBLPromise",
 ]
 
-var foundError = false
+private class ErrorLogger {
```

Comment:
If you wanted you could also pass the error logger to `checkFile`.

The benefit would be that tests could straightforwardly verify that certain inputs resulted in errors and that certain inputs did not without needing to add a way to clear the global error log.

Reply:
Done